SCClassLibrary/Common/Math/SimpleNumber.sc

```diff
@@ -695,20 +695,24 @@ SimpleNumber : Number {
 	// see String:asSecs for complement
 
 	asTimeString { |precision = 0.001, maxDays = 365, dropDaysIfPossible = true|
-		var number, decimal, days, hours, minutes, seconds, mseconds;
+		var number, decimal, days, hours, minutes, seconds, mseconds, isNegative = false;
+		if(this < 0, {
+			isNegative = true;
+		});
```

Comment:
this can be simplified to `var number, decimal, days, hours, minutes, seconds, mseconds, isNegative = this < 0;`